controllers/snapshot/snapshot_adapter.go

```diff
@@ -507,6 +508,11 @@ func (a *Adapter) createMissingReleasesForReleasePlans(application *applicationa
 			}
 			a.logger.Info("Marked Release status automated", "release.Name", newRelease.Name)
 		}
+		//get time HaveAppStudioTestsFinished
```

Comment:
While this would work fine in case there is only one `releasePlan`, I think it would cause issue when there are multiple of them because the latency would be recorded multiple times, once for every release created.

If I understood the [STONEINTG-532](https://issues.redhat.com/browse/STONEINTG-532) SLI correctly, the intent is to measure the time between finishing testing and creating the first release. If that's true, then we'd need to ensure that this is called only on the first pass. @MartinBasti feel free to correct me on this.

Reply:
Something like this might work? @MartinBasti 

firstRelease := true

for _, releasePlan := range *releasePlans {
    // this is the existing code here

    if firstRelease {
        startTime, ok := gitops.GetAppStudioTestsFinishedTime(a.snapshot)
        if ok {
            metrics.RegisterReleaseLatency(startTime)
            firstRelease = false
        }
    }
}

return nil

Reply:
yes, that sound's good

Reply:
done and this flag is tested in the integration_test, where the metric function runs, to ensure only the first release is measured and the flag is working

Reply:
The solution looks good to me, thanks.